### PR TITLE
RD-2078 Code review follow-ups

### PR DESCRIPTION
**Pull request by kubama**
Addresses code review comments from #1340 and #1343

Since there are some refactorings impacting broader context I scheduled system tests build: https://jenkins.cloudify.co/view/UI/job/Stage-UI-System-Test/558/

### CODE REVIEW

**Review by gregcfy — APPROVED** [review]

File: widgets/common/src/UpdateDeploymentModal.jsx [hunk 1]
```diff
@@ -121,7 +118,7 @@ function UpdateDeploymentModal({ open, deploymentId, onHide, toolbox }) {
             .doUpdate(
                 deployment.id,
                 blueprint.id,
-                InputsUtils.getInputsToSend(inputsPlanForUpdate, deploymentInputs, inputsWithoutValue),
+                InputsUtils.getInputsMap(inputsPlanForUpdate, deploymentInputs),
```
Comment (qooban) [inline widgets/common/src/UpdateDeploymentModal.jsx:121]: I don't see `inputsWithoutVaue` after removing `InputsUtils.addErrors`, so it is not necessary here?

**Review by qooban — COMMENTED** [review]
Just one comment-question from my side.

File: widgets/common/src/UpdateDeploymentModal.jsx (hunk 1 above)
Comment (kubama) [reply widgets/common/src/UpdateDeploymentModal.jsx:121]: It's not necessary - in original implementation after calling `getInputsToSend` the value of `inputsWithoutValue` was not used.

Comment (qooban) [reply widgets/common/src/UpdateDeploymentModal.jsx:121]: Got it, approving then.

**Review by qooban — APPROVED** [review]

**Pull request merged**